src/app/main.rs

```diff
@@ -883,17 +883,18 @@ where
     );
     let log = server.log().clone();
 
-    let future = log.future(bound_port.listen_and_fold(
-        (),
-        move |(), (connection, remote_addr)| {
-            let s = server.serve(connection, remote_addr, h2_settings);
+    let future = log.future(
+        bound_port.listen_and_fold((), move |(), (connection, remote)| {
+            let s = server.serve(connection, remote, h2_settings);
+            // .instrument(info_span!("conn", %remote));
```

Comment:
Is this `.instrument()` a TODO or can it be removed?

Reply:
This is a TODO for a future PR that will replace our homebrew log contexts with tokio-trace spans. I'll leave a comment here as well.